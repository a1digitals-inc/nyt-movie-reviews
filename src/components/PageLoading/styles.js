import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
`;

export const Content = styled.div`
  display: flex;
  flex-direction: column;
  text-align: center;
  margin-top: 90px;

  img {
    height: 53px;
  }
`;
