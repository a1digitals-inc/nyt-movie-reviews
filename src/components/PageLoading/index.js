import React from 'react';

import { Container, Content, DotWrapper, Dot } from './styles';

export default function PageLoading() {
  return (
    <Container>
      <Content>
        <DotWrapper>
          <Dot delay="0s" />
          <Dot delay=".1s" />
          <Dot delay=".2s" />
        </DotWrapper>
      </Content>
    </Container>
  );
}
