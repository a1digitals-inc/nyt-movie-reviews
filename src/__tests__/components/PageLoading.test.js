import React from 'react';
import { shallow } from 'enzyme';

import PageLoading from '../../components/PageLoading';

describe('PageLoading component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<PageLoading />);
    expect(wrapper).toMatchSnapshot();
  });
});
