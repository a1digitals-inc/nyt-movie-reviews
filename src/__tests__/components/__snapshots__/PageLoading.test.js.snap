// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoading component should render without crashing 1`] = `
<styled.div>
  <styled.div>
    <Loading />
  </styled.div>
</styled.div>
`;
